src/warc2zim/main.py

```diff
@@ -198,14 +214,38 @@ def __init__(self, args):
             logger.setLevel(logging.INFO)
 
         self.indexed_urls = set({})
-        self.name = args.name
-        if not self.name:
-            self.name, ext = os.path.splitext(args.inputs[0])
-            self.name += ".zim"
+        self.output = args.output
+        if not self.output:
+            self.output, ext = os.path.splitext(args.inputs[0])
+            self.output += ".zim"
 
         self.inputs = args.inputs
         self.replay_viewer_source = args.replay_viewer_source
-        self.main_url = args.main_url
+
+        self.main_url = args.url
+        self.include_all = args.include_all
+        self.include_domains = args.include_domains
+        if self.main_url:
+            if not self.include_all and not self.include_domains:
+                self.include_domains = [urlsplit(self.main_url).netloc]
+
+        self.date = None  # date is loaded from date of main page in WARC
+
+        self.favicon_url = args.favicon
+        self.language = args.lang
+        self.title = args.title
+
+        self.metadata = {
+            "name": args.name,
+            "description": args.desc,
+            "creator": args.creator,
+            "publisher": args.publisher,
+            "tags": ";".join(args.tags) or None,
+            # optional
+            "source": args.source,
+            "flavour": os.path.basename(sys.argv[0]) + " " + " ".join(sys.argv[1:]),
```

Comment:
There is no Flavour for warc2zim files. You should just not set it.

Reply:
set to ""